legend-engine-entitlement/src/main/java/org/finos/legend/engine/entitlement/model/specification/relationalDatabase/RelationalDatabaseTableSpecification.java

```diff
@@ -0,0 +1,47 @@
+//  Copyright 2023 Goldman Sachs
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package org.finos.legend.engine.entitlement.model.specification.relationalDatabase;
+
+import org.finos.legend.engine.entitlement.model.specification.DatasetSpecification;
+
+public class RelationalDatabaseTableSpecification extends DatasetSpecification
```

Comment:
for now, it's being here is fine, we can totally offset this to the store itself though

Reply:
This is a TODO